### PR TITLE
feature/ST-226-security-code-UT

### PR DESCRIPTION
**Pull request by pgs-gmartowski**

### CODE REVIEW

File: test/components/security-code/SecurityCode.spec.ts [hunk 1]
```diff
@@ -166,16 +182,37 @@ describe('SecurityCode', () => {
...
 
   // given
-  describe('onPaste', () => {
-    const { instance } = securityCodeFixture();
+  describe('onPaste()', () => {
+    const { instance } = SecurityCodeFixture();
+    let spyIsMaxLengthReached: jest.SpyInstance;
+    let spySendState: jest.SpyInstance;
+    // @ts-ignore
+    // @TODO clipboard event is not recognizable.
```
Comment (pgs-gmartowski) [inline test/components/security-code/SecurityCode.spec.ts]: @james-simpson-ST Possibly we need to mock Clipboard event cause typescript doesn't properly implemented it (that's only my thought be it looks like that). What I receive in error logs it's ClipboardEvent is not defined.

Comment (pgs-gmartowski) [reply test/components/security-code/SecurityCode.spec.ts]: I just left this case for a moment because of new tasks from todays DS.

Comment (pgs-gmartowski) [reply test/components/security-code/SecurityCode.spec.ts]: same situation is in FormFields.ts and other classes connected with input components

Comment (james-simpson-ST) [reply test/components/security-code/SecurityCode.spec.ts]: Yes I think the other tasks are higher priority so that makes sense :)

**Pull request merged**